src/controllers/accommodation.js

```diff
@@ -340,15 +341,15 @@ class accommodation {
       } = req;
       const pagination = paginate(page, limit);
       const allAccommodations = await Accommodations.findAll({
-        attributes: {
-          exclude: ['createdBy', 'imageUrl', 'country', 'createdAt', 'updatedAt']
-        },
         include: [{
           model: Rooms,
           attributes: {
             exclude: ['accommodationsID', 'createdBy', 'createdAt', 'updatedAt']
```

Comment:
accommodationsID will be needed

Reply:
@blaise82 I just implemented the feedback, thanks for good insight.